site/content/docs/user/configuration.md

```diff
@@ -15,7 +15,9 @@ This guide covers how to configure KIND cluster creation.
 
 We know this is currently a bit lacking right now and will expand it over time - PRs welcome!
 
-<!--TODO: TOC?-->
+1. [Getting Started](#getting-started)
+2. [Cluster-Wide Options](#clusterwide-options)
```

Comment:
I think we should prefer a TOC that won't get stale.

Reply:
Got it. Quick look gives me 
- We can introduce `toc` from hugo side using something like [this](https://gohugo.io/content-management/toc/). But I tried that and it breaks our UI. We would need to bring in CSS changes to accommodate this `toc` ( probably something collapsible either in sidebar or at top of the page). 
- Tried looking at something from markdown side. But that would require adding new plugin. 

Both of this is beyond my knowledge level ( don't even want to try my hand in CSS side of things. :smile: ). Let me know if anything simple can be done.

Reply:
I haven't done this with hugo yet either